### DISCUSSION TITLE
Extension administrator limited?

### DISCUSSION THREAD

**Discussion by StigNil** [solidify/scale#28] [General]
Hi 
Is there a limit on the number of ext admins on scale? 
We are on a business plan.

**Comment by MOlausson** [comment]
Hi @StigNil. All users, regardless of role, are counted against the number of users in the license. There should be no limit on the number of admins as long as the total number of users are less than the license count. What's the scenario for you?

**Reply by StigNil** [reply]
We are under the 100 users, but are experiencing that users that have been changed to the extension administrator role, are not changed. 
This has happened several times.

**Reply by MOlausson** [reply]
@StigNil Can it be that admins try to change roles by adding the users again with a different role? We've found an issue when doing that, then the user will not get the new role. If you instead select the user from the list and manage/change role it works as expected. We've added a task to align the add users flow to also change the role since the general Azure DevOps user management behaves that way. But I hope this explains what you see for now.

/cc @JimmyKB
